### ISSUE TITLE
How do I change language?

### ISSUE THREAD

**Issue by domainpt** [herbix/chunkLauncher#9]
title

**Comment by herbix** [comment]
It reads system languages.
You can add `-Duser.language=en -Duser.country=US` to java arguments to override it.
like:
```sh
java -Duser.language=en -Duser.country=US -jar ChunkLauncher.jar
```

**Comment by K0-RR** [comment]
A GUI option to change language would be nice to have.
Some language-specific characters are not displayed properly.
![Screenshot from 2022-01-27 16-19-51](https://user-images.githubusercontent.com/43277609/151388538-5b636036-76b2-43d0-a6b0-e6dc5a4a0837.png)
![Screenshot from 2022-01-27 16-20-17](https://user-images.githubusercontent.com/43277609/151388593-ba5a4e8d-64bf-4345-84c8-7c92dcf739a8.png)
![Screenshot from 2022-01-27 16-36-24](https://user-images.githubusercontent.com/43277609/151391824-0023f1e5-ee63-4502-97a8-c58b150a219c.png)

Also, @herbix your commit https://github.com/herbix/chunkLauncher/commit/f1ba0cfa944ac426f4e5ce81117cd6cc580d3828 broke Polish translation. I think you wanted to shorten it and it can be done like this:
```
ui.tab.profile=Profil
ui.tab.module=Moduły
```
`I think "Wersje", "Versions" for English would fit better.`
```
ui.tab.directory=Katalog
ui.tab.system=Ustawienia
```

Then in English
```
ui.tab.profile=Profile
ui.tab.module=Modules
ui.tab.directory=Directory
ui.tab.system=Settings
```

Pinging @glowiak as it's their translation. About the translation of "State" - "Tryb" - I think "Status" is better fitting.

**Comment by herbix** [comment]
Thank you @RDKRACZ!

I created a fix of character displaying and wordings for `ui.tab.*` here: https://github.com/herbix/chunkLauncher/commit/0591748a191240f72025c84241e7c2c286d26b75.
You can found latest build here:
https://github.com/herbix/chunkLauncher/actions/runs/1757541156

I'll create a new release if we have a solution of translation of "State".

Regarding language changing GUI option, it needs effort and I'll do it later.

**Comment by herbix** [comment]
Released [ChunkLauncher 1.10.2](https://github.com/herbix/chunkLauncher/releases/tag/1.10.2), you can change language here:
![image](https://user-images.githubusercontent.com/7110965/151594711-6fe28444-7d80-4626-9b86-0cc17f683a39.png)
